opencti-platform/opencti-front/src/private/components/locations/positions/PositionEditionOverview.js

```diff
@@ -299,8 +266,8 @@ class PositionEditionOverviewComponent extends Component {
               name="latitude"
               label={t('Latitude')}
```

Comment:
missing translation for Latitude and Longitude

Reply:
Fixed